### DISCUSSION TITLE
open3e on Arduino?

### DISCUSSION THREAD

**Discussion by bomilkar** [open3e/open3e#168] [Q&A]
Does anybody know of an implementation of open3e on Arduino or some other single chip controller?
I have a script running on a Pi2b to control the room temperature. Now I want to make it fool-proof so that it won't get hurt by power fails etc. In other words: an OS-less implementation.

**Comment by TheSmartGerman** [comment]
No. There where some tries to use ESP32: https://github.com/maromme/esphome-vitoair/

**Comment by MyHomeMyData** [comment]
Another possible approach is "Cannelloni" using an ESP32 as edge device: https://github.com/MyHomeMyData/ESP32cannelloni
